x/bank/types/key.go

```diff
@@ -22,8 +22,9 @@ const (
 
 // KVStore keys
 var (
-	BalancesPrefix = []byte("balances")
-	SupplyKey      = []byte{0x00}
+	BalancesPrefix   = []byte("balances")
+	SupplyKey        = []byte{0x00}
+	DenomMetaDataKey = []byte("denomMetadata")
```

Comment:
Let’s index denom meta by denom. So we can have a function `func DenomMetadataKey(denom string) []byte` instead. Also let’s use a single byte prefix like `0x1`. The key would then be concat(0x1, denom).